cpp/src_prims/stats/minmax.h

```diff
@@ -56,13 +56,13 @@ __host__ __device__ double decode(long long val) {
 }
 
 template <typename T, typename E>
-__device__ T atomicMaxBits(T* address, T val) {
+__device__ __forceinline__ T atomicMaxBits(T* address, T val) {
```

Comment:
Better to use the `DI` macro found in `src_prims/cuda_utils.h`.

Reply:
Done